src/components/ReviewDetail/ReviewContent/ContentHandler.tsx

```diff
@@ -0,0 +1,84 @@
+import { callDeleteReviewDetailAPI } from '@/Api/reviewDetail';
+import { reviewContentState } from '@/store/store';
+import { CREATE_REVIEW_PAGE, USER_PAGE } from '@/utils/constants';
+import { formatDate } from '@/utils/format';
+import { BsTrash } from 'react-icons/bs';
+import { FaRegEdit } from 'react-icons/fa';
+import { useLocation, useNavigate } from 'react-router-dom';
+import { useRecoilValue } from 'recoil';
+
+const ContentHandler = () => {
+  const {
+    state: { id },
+  } = useLocation();
+  const navigate = useNavigate();
+  const { userId, author, title, image, channel, createdAt } =
+    useRecoilValue(reviewContentState);
+
+  const onDeleteReviewDetail = async () => {
+    // TODO 모달 컴포넌트로 변경 예정 @chunwookJoo
+    if (confirm('해당 게시글을 삭제할까요?')) {
+      await callDeleteReviewDetailAPI(id);
+      navigate(`/category/${channel.name}`, {
+        state: { id: channel._id, name: channel.name },
+      });
+    }
+  };
+
+  const updateBodyProp = {
+    title,
+    image,
+    channelId: channel._id,
+  };
+
+  const onMovePage = (page: string) => {
```

Comment:
```suggestion
  const onMovePage = (page: 'userPage' | 'reviewUpdate') => {
```

p3;
타입 안전성을 위해 타입 범위를 좁혀주세요.

Reply:
범위 안좁히면, switch default문 추가해주세용